spacy/tests/lang/da/test_noun_chunks.py

```diff
@@ -0,0 +1,16 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import pytest
+
+
+def test_noun_chunks_is_parsed(da_tokenizer):
+    """Test that noun_chunks raises Value Error for 'da' language if Doc is not parsed.
+    To check this test, we're constructing a Doc
+    with a new Vocab here and forcing is_parsed to 'False'
+    to make sure the noun chunks don't run.
+    """
+    doc = da_tokenizer("Det er en sætning")
+    doc.is_parsed = False
+    with pytest.raises(ValueError):
+        list(doc.noun_chunks)
```

Comment:
Could you add a few tests with example sentences with different kinds of noun chunks to make sure everything is working as intended? Swedish is a good example again here: https://github.com/explosion/spaCy/blob/bc027dc35c7b2e4482561ca8a818a9b8904b4984/spacy/tests/lang/sv/test_noun_chunks.py

Since the tests don't have access to models like `da_core_news_sm`, you have to construct the docs by hand that are used for the tests. The `get_doc` function uses relative heads rather than absolute heads, which you can calculate with `heads = [t.head.i - t.i for t in doc]` from an existing doc.

Reply:
I added some sentences in the same type of test as the swedish one.